src/dataprotection/azext_dataprotection/manual/custom.py

```diff
@@ -180,12 +181,16 @@ def dataprotection_backup_instance_initialize(datasource_type, datasource_id, da
     policy_parameters = None
     # Azure Disk specific code for adding datastoreparameter list in the json
     if datasource_type == "AzureDisk":
+        if snapshot_resource_group_name is None:
+            raise CLIError("--snapshot-resource-group-name must be provided when initializing AzureDisk workload")
```

Comment:
This is a breaking change. Can we have a default `resource_group` for old customers not providing `--snapshot-resource-group-name`?